macros/base/probing/generic_probe.cfg

```
@@ -0,0 +1,69 @@
+# This file is used as in interface to activate/deactivate every probe type
+# depending of the needs:
+#  - TAP: need to be at a correct nozzle temperature to avoid burning the PEI when used
+#  - Dockable probe: need to be attached/docked when used
+#  - ... can be improved depending of new probes needs ...
+
+[gcode_macro ACTIVATE_PROBE]
+description: Put the machine in a state being able to probe
+variable_temperature: 0
+gcode:
+    {% set LOCK = params.LOCK|default(False, boolean=true) %}
+
+    {% set probe_type_enabled = printer["gcode_macro _USER_VARIABLES"].probe_type_enabled %}
+    {% set tap_max_probing_temp = printer["gcode_macro _USER_VARIABLES"].tap_max_probing_temp|float %}
+
+    # If a dockable probe is defined, then check and ATTACH the probe
+    {% if probe_type_enabled == "dockable" %}
+        _CHECK_PROBE action=query
+        {% if LOCK %}
+            _ATTACH_PROBE_LOCK
+        {% else %}
+            _ATTACH_PROBE
+        {% endif %}
+    
+    # If a Voron TAP probe is defined, then check the temperature and lower it if needed
+    {% elif probe_type_enabled == "vorontap" %}
+        {% set ACTUAL_TEMP = printer.extruder.temperature %}
+        {% set TARGET_TEMP = printer.extruder.target %}
+
+        SET_GCODE_VARIABLE MACRO=ACTIVATE_PROBE VARIABLE=temperature VALUE={TARGET_TEMP}
+
+        {% if TARGET_TEMP > tap_max_probing_temp %}
+            { action_respond_info('Extruder temperature target of %.1fC is too high for TAP probing, lowering to %.1fC' % (TARGET_TEMP, PROBE_TEMP)) }
```

Hey guys, just passing by on this PR to point to a small error in this MACRO:
```suggestion
            { action_respond_info('Extruder temperature target of %.1fC is too high for TAP probing, lowering to %.1fC' % (TARGET_TEMP, tap_max_probing_temp)) }
```